doc/tutorials/introduction/documenting_opencv/documentation_tutorial.markdown

```diff
@@ -574,7 +628,7 @@ Write the tutorial {#tutorial_documentation_steps_tutorial}
     <em>"table_of_content_*.markdown"</em> file with the needed table and place new record in it
     similar to existing ones.
     @verbatim
--   @subpage tutorial_windows_visual_studio_image_watch
+-   @subpage_info{tutorial_windows_visual_studio_image_watch}
```

Comment:
subpage_info{1}="@subpage \1 @add_page_identifier{\1} @htmlinclude subpage_info.html"

subpage_info is a macro that recalls subpage and adds the information about the toggle options available.